pkg/agent/openflow/pipeline_windows.go

```diff
@@ -21,71 +21,75 @@ import (
 	"net"
 
 	"antrea.io/antrea/pkg/agent/config"
-	"antrea.io/antrea/pkg/agent/openflow/cookie"
 	binding "antrea.io/antrea/pkg/ovs/openflow"
 )
 
-// hostBridgeUplinkFlows generates the flows that forward traffic between the
-// bridge local port and the uplink port to support the host traffic with
-// outside.
-func (c *client) hostBridgeUplinkFlows(localSubnetMap map[binding.Protocol]net.IPNet, category cookie.Category) (flows []binding.Flow) {
-	flows = []binding.Flow{
-		ClassifierTable.BuildFlow(priorityNormal).
+// hostBridgeUplinkFlows generates the flows that forward traffic between the bridge local port and the uplink port to
+// support the host traffic with outside.
+func (f *featurePodConnectivity) hostBridgeUplinkFlows(localSubnetMap map[binding.Protocol]net.IPNet) []binding.Flow {
+	cookieID := f.cookieAllocator.Request(f.category).Raw()
+	flows := f.hostBridgeLocalFlows()
+	flows = append(flows,
+		// This generates the flow to forward ARP packets from uplink port to bridge local port since uplink port is set
```

Comment:
In Windows, flood is disabled, ARP from local Node via bridge port should be output to uplink port.
In Linux, flood is enabled, and ARP from local Node via bridge port can be processed with action `normal` in pipeline.

Reply:
I don't get the causal relationship, could you explain more?

Reply:
In Windows, ARP from Node can be only output to uplink port directly since OVS flood is disabled, then ARP packet can be received by other hosts in the Node subnet. If output ARP with action `normal`, the ARP packets cannot output on uplink port.

In Linux, ARP from Node can be flooded to every port on OVS including uplink port since OVS flood is enabled, then ARP packets can be received by other hosts in the Node subnets.